lib/vre/ch31_form.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+require_relative 'service'
+require_relative 'errors/ch31_errors'
+require 'sentry_logging'
+
+module VRE
+  class Ch31Form < VRE::Service
+    include SentryLogging
+    configuration VRE::Configuration
+    STATSD_KEY_PREFIX = 'api.vre'
+
+    # The Ch31Form class is the means by which the Ch31 aka 28-1900 form is submitted to VR&E
```

Comment:
If this is meant to be the class-level doc it can go on line 7 ( right above `class Ch31Form < VRE::Service` )